.jenkins

```diff
@@ -385,6 +385,64 @@ pipeline {
                         }
                     }
                 }
+
+                stage('SYCL') {
+                    agent {
+                        dockerfile {
+                            filename "Dockerfile.sycl"
+                            dir "docker"
+                            args '-v /tmp/ccache.kokkos:/tmp/ccache'
+                            label 'NVIDIA_Tesla_V100-PCIE-32GB && nvidia-docker'
+                        }
+                    }
+                    steps {
+                        sh 'ccache --zero-stats'
+                        sh 'rm -rf build && mkdir -p build'
+                        dir('build') {
+                            sh '''
+                                cmake \
+                                    -D CMAKE_INSTALL_PREFIX=$ARBORX_DIR \
+                                    -D CMAKE_BUILD_TYPE=Release \
+                                    -D CMAKE_CXX_COMPILER=clang++ \
+                                    -D CMAKE_CXX_EXTENSIONS=OFF \
+                                    -D CMAKE_CXX_FLAGS="-Wpedantic -Wall -Wextra -Wno-unknown-cuda-version -fsycl -fsycl-targets=nvptx64-nvidia-cuda-sycldevice" \
```

Comment:
Do you need the `-Wno-unknown-cuda-version` with CUDA 10.2?

Reply:
Yes, it says the last supported version is 10.1.